src/components/cluster/detail/certificate_orgs_label.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { shallow } from 'enzyme';
+
+import CertificateOrgsLabel from './certificate_orgs_label.js';
+
+it('renders single label without crashing', () => {
```

Comment:
the message is a bit weird IMO, `without crashing` can be omit and it will be  clear enough

Reply:
I do not know why but people started calling the first basic render test for react components 'renders without crashing'. Maybe because 'renders' is not a long enough.

a different and more fancy version would be 'renders without exploding' :)